lib/request.js

```diff
@@ -233,11 +242,25 @@ Request.prototype.sendResponse = function(response) {
 
   response.writeHead(this.response.statusCode, headers);
 
-  if (this.response.filePath) {
-    var readStream = fs.createReadStream(this.response.filePath);
-    readStream.pipe(response);
+  if (isStream(this.response.body)) {
+    // Because we may need to respond with this body more than once, if it is a stream,
+    // we make a buffer copy and use that as the body for future responses.
```

Comment:
I added a new PR #26 that only saves the stream contents when actually needed.  So with that, the rule is just "don't use multiple responses for giant stream/files if you don't have room to store a copy of the stream", which is probably reasonable.